CHANGELOG.md

```diff
@@ -4,6 +4,34 @@ All notable changes to this project will be documented in this file.
 Each new release typically also includes the latest modulesync defaults.
 These should not affect the functionality of the module.
 
+## [v3.1.0](https://github.com/voxpupuli/puppet-mongodb/tree/v3.1.0) (2020-02-10)
+
+[Full Changelog](https://github.com/voxpupuli/puppet-mongodb/compare/v3.0.0...v3.1.0)
+
+**Implemented enhancements:**
+
+- restrict opsmanager conf file permissions for better security [\#563](https://github.com/voxpupuli/puppet-mongodb/pull/563) ([FLiPp3r90](https://github.com/FLiPp3r90))
+
+**Fixed bugs:**
+
+- Wrong APT-key [\#546](https://github.com/voxpupuli/puppet-mongodb/issues/546)
+- Mongo 4.0.x: unable to create user [\#525](https://github.com/voxpupuli/puppet-mongodb/issues/525)
+- user creation idempotency issues [\#412](https://github.com/voxpupuli/puppet-mongodb/issues/412)
+- Fixed the problem: the user was not created for Mongodb 4.x [\#561](https://github.com/voxpupuli/puppet-mongodb/pull/561) ([identw](https://github.com/identw))
+- Only create database and user when mongodb\_is\_master [\#558](https://github.com/voxpupuli/puppet-mongodb/pull/558) ([JvGinkel](https://github.com/JvGinkel))
+- Fix output of functions gathering replset info. [\#556](https://github.com/voxpupuli/puppet-mongodb/pull/556) ([czernitko](https://github.com/czernitko))
+
+**Closed issues:**
+
+- Puppet error "Unknown function: 'mongodb\_password'" [\#569](https://github.com/voxpupuli/puppet-mongodb/issues/569)
```

Comment:
I always wonder about these. This was a user error in their setup and no code was changed in this module. Normally I skip these but I wonder what others think.

Reply:
I've applied the 'skip-changelog' label.